client-spark/common/src/main/java/org/apache/uniffle/shuffle/manager/RssShuffleManagerBase.java

```diff
@@ -807,6 +808,17 @@ public MutableShuffleHandleInfo reassignOnBlockSendFailure(
     }
   }
 
+  @Override
+  public void stop() {
+    if (shuffleManagerClient != null) {
```

Comment:
@zuston I make the lifecycle of `ShuffleManagerClient` with ShuffleManage, referencee with `ShuffleWriteClient`.